ROS/osr_control/osr_control/roboclaw_wrapper.py

```diff
@@ -420,16 +421,94 @@ def stop_motors(self):
 
     def read_errors(self):
         """Checks error status of each motor controller, returns 0 if no errors reported"""
-        err = [0] * 3
+        err = ['0'] * 5
         for i in range(len(self.address)):
             err_int = self.rc.ReadError(self.address[i])[1]
-            # convert to hexadecimal and then to string for easy decoding
-            err[i] = str(hex(err_int))
+
             if err_int != 0:
-                self.log.error(f"Motor controller '{self.address[i]}' reported error code {err[i]}")
-        
+                # convert to hexadecimal and then to string for easy decoding
+                err[i] = str(hex(err_int))
+
+                errString, hasError = self.decode_error(err_int)
```

Comment:
```suggestion
                err_string, has_error = self.decode_error(err_int)
```

Follow [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names)

Reply:
See commit ede2052

Reply:
And commit e1cdd66